api.js

```diff
@@ -18,9 +19,10 @@ api.use(express.json())
 api.use("/api/v1/", todoAPI)
 api.use("/api/v1/", checklistAPI)
 api.use("/api/v1/", userAPI)
+api.use("/api/v1/", teamAPI)
 
 //Index
-api.get("/", (req, res)=>{
+api.get("/", async (req, res)=>{
     console.log("INDEX")

```

Comment:
Remove console.log

Reply:
And why it be async function?

Reply:
once use it as a command testing function and forget to delete 'async'​ out of it, we will fix it next commit. :)​

Reply:
Ok. You can push to `feature/teamAPI` branch, don't need to create new PR na.